### PR TITLE
Karbon docs

### PR DESCRIPTION
**Pull request by coderGo93**
Added docs for resource and datasource(s):
- Cluster
- Clusters
- Private registry
- Private Registries

### CODE REVIEW

**Comment by marinsalinas** [comment]
@yannickstruyf3 can you take a look?

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: explanation: 

Maximum wait time for the Karbon cluster to provision.

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: Storage class configuration attribute for defining the persistent volume attributes.

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: Allows the Karbon cluster to pull images of a list of private registries

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: * `master_node_pool `: - Configuration of the master node pools.

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: Pins the persistent volumes to the flash tier in case of a `true` value.

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: The password

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: Name of

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: Username of..

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: Name of the private registry to add to the Karbon cluster

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: List of private registries

Note: Should we rename the parameter to make it clear that it is a list?

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: VM configuration in AHV

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: List of the deployed nodes in the node pool

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: Hostname of the deployed node

File: website/docs/d/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/d/karbon_cluster.html.markdown]: IP of the deployed node

File: website/docs/r/karbon_cluster.html.markdown
Comment (yannickstruyf3) [inline website/docs/r/karbon_cluster.html.markdown]: Not optional?

**Review by yannickstruyf3 — CHANGES_REQUESTED** [review]
I added some remarks with regards to the documentation. Please review the comments and apply them to the rest of the Karbon documentation files.
Thanks

File: website/docs/d/karbon_cluster.html.markdown
Comment (coderGo93) [reply website/docs/d/karbon_cluster.html.markdown]: @marinsalinas should we?

**Comment by marinsalinas** [comment]
@yannickstruyf3 @gabybeitler  can you give a second review?

**Review by marinsalinas — APPROVED** [review]
LGTM! just we need the approval from Nutanix (@gabybeitler or @yannickstruyf3 )

**Pull request merged**
